### PR TITLE
restyle menu

### LINKED ISSUE #23: Redesign Menu

### PR DESCRIPTION
**Pull request by aileenpongnon**
resolves #23 

- slower transition on menu link hover
- adjusted sizing of the texts in our menu

### CODE REVIEW

**Comment by habraham2023** [comment]
Not sure why but the fade out effect is still too slow for my liking.

File: src/components/Menu.js [hunk 1]
```diff
@@ -9,49 +12,133 @@ import { Link } from "react-router-dom";
...
+          open={open}
+          onClose={() => setOpen(false)}
+        >
+          <div className="min-h-screen">
+            <Transition.Child
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0"
+              enterTo="opacity-100"
+              leave="ease-in duration-700"
```
Comment (habraham2023) [inline src/components/Menu.js:45]: The duration of the fade out should NOT be 700 ms. That is completely too long and unreasonable. I would try values between 200 and 300 ms and see which ones work best.

**Review by habraham2023 — CHANGES_REQUESTED** [review]
As mentioned before I would make the fade out effect quicker as it feels laggy in its current state.

**Comment by aileenpongnon** [comment]
> As mentioned before I would make the fade out effect quicker as it feels laggy in its current state.

nah it broke again, i'm gonna @ you when its done! my revert messed up and kept the lagging code instead of the fixed one so disregard

File: src/components/Menu.js (hunk 1 above)
Comment (aileenpongnon) [reply src/components/Menu.js:45]: lmao this wasn't intentional, my replace-all overwrote all the duration-200 lines! was planning to change this part back to how it was originally

**Comment by habraham2023** [comment]
LGTM

**Review by habraham2023 — APPROVED** [review]

**Pull request merged**